packages/ui/src/components/select/SignerSelection.tsx

```diff
@@ -68,35 +114,8 @@ const SignerSelection = ({ className, possibleSigners, onChange }: Props) => {
       className={className}
       options={signersList}
       filterOptions={filterOptions}
-      renderOption={(props, option, index) => (
-        <Box
-          component="li"
-          sx={{
-            mr: '.5rem',
-            pt: '.8rem !important',
-            pl: '1.5rem !important',
-            flexShrink: 0
-          }}
-          {...props}
-          key={option?.address}
-        >
-          <AccountDisplay address={option?.address || ''} />
-        </Box>
-      )}
-      renderInput={(params) => (
-        <TextField
-          {...params}
-          label="Signing with"
```

Comment:
> I would prefer that issue focus on one subject, otherwise, it's not enjoyable to work in one huge PR

Regarding this in particular, I totally agree. Please open an issue and we can tackle it in a dedicated PR.

The bug I mentioned here should however be fixed before we merge https://github.com/ChainSafe/Multix/pull/235#discussion_r1262680318

Reply:
@Tbaut the bug you mentioned is fixed